scripts/git/version.sh

```diff
@@ -37,14 +37,14 @@ LAST_VERSION_TAG=$(git describe --abbrev=1 --tags --match="${UPSTREAM_ORG}/${COM
 LAST_VERSION=${LAST_VERSION_TAG##*/}
 #echo "LAST_VERSION: $LAST_VERSION"
 
+# Current commit
+COMMIT=$(git rev-parse --short HEAD)
+
 # If no version was found, use default v0.0.0.
 if [[ -z "$LAST_VERSION" ]]; then
-  LAST_VERSION="v0.0.0"
+  LAST_VERSION="v0.0.0-$COMMIT"
```

Comment:
Should the `if [[ "$LAST_VERSION" == *"-"* ]]; then` be an else-if clause to this block?  It seems like maybe we're setting LAST_VERSION to a value which will trigger that second block so we're adding the commit, then removing the commit, then adding the commit (if I read things correctly)

Reply:
Good catch, thank you.